sdk/cosmosdb/cosmos/src/request/SharedOptions.ts

```diff
@@ -23,7 +23,6 @@ export interface SharedOptions {
    */
   abortSignal?: AbortSignal;
   /**
-   * **(Beta - There may be breaking changes in this property in future releases)**
    * Sets the staleness value associated with the request in the Azure CosmosDB service. For requests where the {@link
    * com.azure.cosmos.ConsistencyLevel} is {@link com.azure.cosmos.ConsistencyLevel#EVENTUAL}, responses from the
```

Comment:
Consistency level can be either eventual or session

Reply:
done.